packages/web3-eth/package.json

```diff
@@ -40,7 +40,8 @@
 		"prettier": "^2.4.1",
 		"ts-jest": "^27.0.7",
 		"typescript": "^4.5.2",
-		"web3-providers-http": "4.0.0-alpha.1"
+		"web3-providers-http": "4.0.0-alpha.1",
+		"web3-eth-abi": "4.0.0-alpha.0"
```

Comment:
Added `web3-eth-abi` dev dependency to use `decodeParameters` in tests like [eth_call](https://github.com/ChainSafe/web3.js/blob/ad85e4256d2ee6017732c2c48b077d706c3cdde8/packages/web3-eth/test/integration/web3_eth/call.test.ts#L96)